Sources/PostgreSQL/Database/PostgreSQLDatabaseConfig.swift

```diff
@@ -29,4 +30,25 @@ public struct PostgreSQLDatabaseConfig {
         self.database = database
         self.password = password
     }
+
+    public init(connectionString: String) throws {
```

Comment:
param label can probably just be `url` here.
also a doc comment with example would be awesome